### PR TITLE
Add cluster networkpool option

### PR DESCRIPTION
**Pull request by paurosello**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: cmd/template/cluster/flag.go [hunk 1]
```diff
@@ -73,6 +75,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&f.ExternalSNAT, flagExternalSNAT, false, "AWS CNI configuration.")
 	cmd.Flags().StringVar(&f.PodsCIDR, flagPodsCIDR, "", "CIDR used for the pods.")
 	cmd.Flags().StringVar(&f.Credential, flagCredential, "credential-default", "Cloud provider credentials used to spin up the cluster.")
+	cmd.Flags().StringVar(&f.NetworkPool, flagNetworkPool, "", "NetworkPool for the subnets of the nodes.")
```
Comment (xh3b4sd) [inline cmd/template/cluster/flag.go:78]: Should we add some example so it is easier for at least me to picture that? What would be a legitimate example? 😄 

```suggestion
	cmd.Flags().StringVar(&f.NetworkPool, flagNetworkPool, "", "NetworkPool for the subnets of the nodes, e.g. 127.0.0.1/16.")
```

**Review by xh3b4sd — APPROVED** [review]

File: cmd/template/cluster/flag.go (hunk 1 above)
Comment (paurosello) [reply cmd/template/cluster/flag.go:78]: This flag references a "name" so maybe I can just say that in the help "Name of the NetworkPool that will be used for the subnets of the nodes"

**Comment by marians** [comment]
Please also update https://github.com/giantswarm/docs/blob/master/src/content/reference/kubectl-gs/template-cluster.md once this here is official. Thanks!

**Comment by tfussell** [comment]
Closing as there has been no activity for over a year. Feel free to reopen.

**Pull request closed without merging**